### DISCUSSION TITLE
Save difficulty adjust mod settings to each beatmap

### DISCUSSION THREAD

**Discussion by LOKSTED** [ppy/osu#18351] [Ideas]
Currently the mod applies the adjusted difficulty settings to every beatmap and resets when it is turned off.
What do you think about the idea of saving the settings to each beatmap so that it is applied whenever the song is played again with the mod on.

^That sounds confusing so here is an example:
If I play a song with the approach rate turned to 9, then turn the mod off or quit the game, the mod should set the approach rate to 9 again whenever it is turned on.

This would be helpful to those who want to permanently adjust the difficulty settings without having to edit the map and lose the scoreboard

**Comment by frenzibyte** [comment]
I think this could work much better with the concept of "mod presets" instead, in which you may *potentially* have the ability to have certain mods on certain beatmaps selected automatically.
